pkg/networkutils/ippool.go

```diff
@@ -0,0 +1,60 @@
+package networkutils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+type IPPool []string
+
+func NewIPPool() IPPool {
+	return IPPool{}
+}
+
+func NewIPPoolFromString(fromString string) IPPool {
+	return IPPool(strings.Split(fromString, ","))
+}
+
+func NewIPPoolFromEnv(ipPoolEnvVar string) (IPPool, error) {
+	value, ok := os.LookupEnv(ipPoolEnvVar)
+	if !ok {
+		return NewIPPool(), fmt.Errorf("%s environment ip pool does not exist", ipPoolEnvVar)
+	}
+	if value != "" {
+		return NewIPPoolFromString(value), nil
+	}
+	return NewIPPool(), nil
```

Comment:
should we return an error here if the IPPool is empty?

Reply:
I figured we may have a need to support empty envvar. LookupEnv supports empty env varas as well. For instance, you may want to take a different action if the var exists but is empty vs if it does not exist all.